nativescript-angular/http/ns-http.ts

```diff
@@ -14,78 +14,78 @@ import "rxjs/add/observable/fromPromise";
+    }
 
-  /**
-   * Performs a request with `request` http method.
-   */
-  request(req: string | Request, options?: RequestOptionsArgs): Observable<Response> {
-    const urlString = typeof req === "string" ? req : req.url;
-    if (isLocalRequest(urlString)) {
-      return this._requestLocalUrl(urlString);
-    } else {
-      return super.request(req, options);
+    /**
+     * Performs a request with `request` http method.
+     */
+    request(req: string | Request, options?: RequestOptionsArgs): Observable<Response> {
+        const urlString = typeof req === "string" ? req : req.url;
+        if (isLocalRequest(urlString)) {
+            return this._requestLocalUrl(urlString);
+        } else {
+            return super.request(req, options);
+        }
     }
-  }
 
-  /**
-   * Performs a request with `get` http method.
-   */
-  get(url: string, options?: RequestOptionsArgs): Observable<Response> {
-    if (isLocalRequest(url)) {
-      return this._requestLocalUrl(url);
-    } else {
-      return super.get(url, options);
+    /**
+     * Performs a request with `get` http method.
+     */
+    get(url: string, options?: RequestOptionsArgs): Observable<Response> {
+        if (isLocalRequest(url)) {
+            return this._requestLocalUrl(url);
+        } else {
+            return super.get(url, options);
+        }
     }
-  }
 
-  /**
-   * Uses a local file if `~/` resource is requested.
-   * @param url
-   */
-  private _requestLocalUrl(url: string): Observable<Response> {
-    // normalize url
-    url = normalizeLocalUrl(url);
-    // request from local app resources
-    return Observable.fromPromise<Response>(new Promise((resolve, reject) => {
-      let app = this.nsFileSystem.currentApp();
-      let localFile = app.getFile(url);
-      if (localFile) {
-        localFile.readText().then((data) => {
-          resolve(responseOptions(data, 200, url));
-        }, (err: Object) => {
-          reject(responseOptions(err, 400, url));
-        });
-      } else {
-        reject(responseOptions("Not Found", 404, url));
-      }
-    }));
-  }
+    /**
+     * Uses a local file if `~/` resource is requested.
+     * @param url
+     */
+    private _requestLocalUrl(url: string): Observable<Response> {
+        url = normalizeLocalUrl(url);
+
+        // request from local app resources
+        return Observable.fromPromise<Response>(new Promise((resolve, reject) => {
+            let app = this.nsFileSystem.currentApp();
+            let localFile = app.getFile(url);
+            if (localFile) {
+                localFile.readText().then((data) => {
+                    resolve(responseOptions(data, 200, url));
+                }, (err: Object) => {
+                    reject(responseOptions(err, 400, url));
+                });
+            } else {
+                reject(responseOptions("Not Found", 404, url));
+            }
+        }));
+    }
 }
 
 function isLocalRequest(url: string): boolean {
```

Comment:
This is duplicated in `NsHttpBackEnd`. May be a good idea to extract it in a common module?